controllers/articles.js

```diff
@@ -0,0 +1,13 @@
+const app = require("../db/app");
+const { fetchArticleById } = require('../models/articles')
+
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    fetchArticleById(article_id).then((rows) =>{
```

Comment:
We have control over the parameter within our .then block here. If you renamed rows to be article, you wouldn't have to reassign it on line 7